models.py

```diff
@@ -1136,6 +1136,12 @@ class FeatureForm(forms.Form):
                               'statuses In developer trial and Origin trial this '
                               'MUST be blank.')
 
+SUMMARY_PLACEHOLDER_TXT = (
```

Comment:
The travis build is failing because this line needs to be indented more to align with the other constants.

Reply:
I fixed it. I hope.